### PR TITLE
Rename GI Classes

### PR DESCRIPTION
**Pull request by reduz**
* **GIProbe** is now **VoxelGI**
* **BakedLightmap** is now **LightmapGI**

As Godot adds more ways to provide GI (as an example, SDFGI in 4.0, and likely more in the future, such as raytraced), the previous existing techniques need more specific names to allow discerning them.

### CODE REVIEW

File: editor/editor_node.cpp [hunk 1]
```diff
@@ -484,8 +484,8 @@ void EditorNode::_update_from_settings() {
 	RS::get_singleton()->environment_set_sdfgi_frames_to_converge(frames_to_converge);
 	RS::EnvironmentSDFGIRayCount ray_count = RS::EnvironmentSDFGIRayCount(int(GLOBAL_GET("rendering/global_illumination/sdfgi/probe_ray_count")));
 	RS::get_singleton()->environment_set_sdfgi_ray_count(ray_count);
-	RS::GIProbeQuality gi_probe_quality = RS::GIProbeQuality(int(GLOBAL_GET("rendering/global_illumination/gi_probes/quality")));
-	RS::get_singleton()->gi_probe_set_quality(gi_probe_quality);
+	RS::VoxelGIQuality voxel_gi_quality = RS::VoxelGIQuality(int(GLOBAL_GET("rendering/global_illumination/voxel_gis/quality")));
```
Comment (clayjohn) [inline editor/editor_node.cpp:487]:
```suggestion
	RS::VoxelGIQuality voxel_gi_quality = RS::VoxelGIQuality(int(GLOBAL_GET("rendering/global_illumination/voxel_gi/quality")));
```

This one might be an issue throughout the engine. It looks like you search and replaced "gi_probe" with "voxel_gi" however, at some places we had "gi_probes". So now you need to find and replace "voxel_gis" with "voxel_gi"

Comment (reduz) [reply editor/editor_node.cpp:487]: Good catch. I will rename voxel_gis to voxel_gi_instances and in the configuration, voxel_gis to voxel_gi.

**Comment by akien-mga** [comment]
On RC we suggested that `SDFGI` could be renamed `DistanceFieldGI` if we want to make the name slightly easier to read/understand, at the cost of verbosity.

**Comment by reduz** [comment]
@akien-mga I think naming is ok as is, VoxelGI should probably be VoxelConeTracedGI, but I think shorter but still meaningful names are better for this stuff. All that matters is that users can distinguish one technique from the other.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**